<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>رقم هاتف الاب:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAddingStudent">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>580</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(194, 64, 0);
color:rgb(255, 255, 255);
font: 15pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="text">
    <string>إلغــــــاء</string>
   </property>
  </widget>
  <widget class="QPushButton" name="labAddStudent">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(223, 221, 220);
color:rgb(0, 0, 0);

font: 18pt &quot;Mohammad Annoktah&quot;;</string>
   </property>
   <property name="text">
    <string>إضافة طالب جديد</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveStudent">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>580</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 170, 127);
color:rgb(255, 255, 255);
font: 15pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="text">
    <string>حفـــــظ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>تاريخ الميلاد</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateStudentDOB">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="locale">
    <locale language="Arabic" country="Yemen"/>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>اللقب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم الثاني</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم الثالث</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtStudentThirName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtStudentSecName">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>190</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم الاول</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtStudentLName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtStudentFName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtStudentParentPhone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDialableCharactersOnly|Qt::ImhDigitsOnly|Qt::ImhMultiLine</set>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="combClasses">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الفصل</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtStudentFName</tabstop>
  <tabstop>txtStudentSecName</tabstop>
  <tabstop>txtStudentThirName</tabstop>
  <tabstop>txtStudentLName</tabstop>
  <tabstop>txtStudentParentPhone</tabstop>
  <tabstop>dateStudentDOB</tabstop>
  <tabstop>labAddStudent</tabstop>
  <tabstop>btnCancelAddingStudent</tabstop>
  <tabstop>btnSaveStudent</tabstop>
  <tabstop>combClasses</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
